pennylane/fourier/utils.py

```diff
@@ -42,3 +42,73 @@ def to_dict(coeffs):
     nvecs = product(*n_ranges)
 
     return {nvec: coeffs[nvec] for nvec in nvecs}
+
+
+def _get_spectrum(op, decimals):
```

Comment:
Here, the option to choose the `decimals` to which to round the frequencies is new. It was previously fixed to `decimals=8`.

Reply:
Fair question, I think not! Will revert.